diff --git a/platform/linux_platform.go b/platform/linux_platform.go
index 54f4dd7..f0918aa 100644
--- a/platform/linux_platform.go
+++ b/platform/linux_platform.go
@@ -747,7 +747,7 @@ func (p linux) setupRunDir(sysDir string) error {
 			return bosherr.WrapErrorf(err, "Making %s dir", runDir)
 		}
 
-		err = p.diskManager.GetMounter().Mount("tmpfs", runDir, "-t", "tmpfs", "-o", "size=1m")
+		_, _, _, err = p.cmdRunner.RunCommand("mount", "tmpfs", runDir, "-t", "tmpfs", "-o", "size=1m")
 		if err != nil {
 			return bosherr.WrapErrorf(err, "Mounting tmpfs to %s", runDir)
 		}
